core/controllers/reader.py

```diff
@@ -21,7 +21,7 @@
 import jinja2
 
 from core.controllers import base
-from core.domain.classifier import lda_string_classifier
+from core.domain.classifier import classifier_registry
```

Comment:
I suggest putting classifier_registry in core.domain (since we're moving all the other stuff to extensions) and getting rid of the classifiers folder.